.eslintrc.json

```
@@ -0,0 +1,27 @@
+{
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true,
+      "modules": true,
+      "ecmaVersion": 6,
+      "experimentalObjectRestSpread": true
+    }
```

This seems off, for a handful of reasons. I'd suggest something like 

```suggestion
  "parserOptions": {  
    "ecmaVersion": 5
```